AutomaticQC/SpikeClassifiers/imosSpikeClassifierNonBurstSavGolOTSU.m

```diff
@@ -51,47 +51,49 @@
 %
 
 if nargin == 1
-    window = 5; %minimum filtering
+    window = 3; %minimum filtering
     pdeg = 2; %quadratic polynomial
-    nbins = 256; % default otsu bins
+    nbins = 100; % default otsu bins
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 elseif nargin == 2
     pdeg = 2; %quadratic polynomial
-    nbins = 256; % default bins in despiking1
+    nbins = 100; % default bins in despiking1
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 elseif nargin == 3
-    nbins = 256; % default bins in despiking1
+    nbins = 100; % default bins in despiking1
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 elseif nargin == 4
     oscale = 1; %arbitrary scale used to reduce the otsu threshold
 end
 
-if window <= 3
-    error('Window need to be even and larger than 3')
-else
-
-    if rem(window, 2) == 0
-        window = window - 1;
-    end
+if window <= 2
+    error('Window need to be larger than 2')
```

Comment:
> again where are these values found? is it just through testing?

The handling here of the window is a bit funky because SavGol filter only accepts odd windows. The defaults were set by a mix of "minimal effect" and "usability". 

In fact, the default at the end of this PR is 5 to be consistent with the behaviour of the filter. A window==3 actually did not change the value because of the implementation.